bin/worker.js

```diff
@@ -0,0 +1,9 @@
+import { reportRequestMetric } from '../lib/influx.js'
+
+export default {
+  async fetch(request, env, ctx, { reportRequestMetric } = { reportRequestMetric }) {
```

Comment:
```suggestion
import { reportRequestMetric as defaultReportRequestMetric } from '../lib/influx.js'

export default {
  async fetch(request, env, ctx, { reportRequestMetric = defaultReportRequestMetric } = {}) {
```


Otherwise, if you call `fetch(request, env, ctx, {})`, `reportRequestMetric` will be undefined (you only provided a fallback for when no options arguement is passed)